heartbeat/scheduler/scheduler.go

```diff
@@ -77,13 +79,33 @@ type Schedule interface {
 	RunOnInit() bool
 }
 
+func getJobLimitSem(jobLimitByType map[string]config.JobLimit) map[string]*semaphore.Weighted {
+	jc := map[string]config.JobLimit{
```

Comment:
I think we can remove this, we don't need limits for non-configured types.

If there's a missing limit we can just make `Acquire` and `Release` noops, which already appears to be the case.

Reply:
Done.